src/ray/object_manager/plasma/store.cc

```diff
@@ -924,14 +913,8 @@ Status PlasmaStore::ProcessMessage(const std::shared_ptr<Client> &client,
     return Status::Disconnected("The Plasma Store client is disconnected.");
     break;
   case fb::MessageType::PlasmaSetOptionsRequest: {
-    std::string client_name;
-    int64_t output_memory_quota;
-    RAY_RETURN_NOT_OK(
-        ReadSetOptionsRequest(input, input_size, &client_name, &output_memory_quota));
-    client->name = client_name;
-    bool success = eviction_policy_.SetClientQuota(client.get(), output_memory_quota);
-    RAY_RETURN_NOT_OK(SendSetOptionsReply(
-        client, success ? PlasmaError::OK : PlasmaError::OutOfMemory));
+    RAY_LOG(WARNING) << "Set client quota deprecated.";
```

Comment:
Maybe removing the whole code path for this?

Reply:
![image](https://user-images.githubusercontent.com/837180/124669861-f3e80900-de67-11eb-91f5-2c4c9f75fb52.png)
yes sir.